### PR TITLE
Initialize repo with required files

### PR DESCRIPTION
**Pull request by bjankord**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra"
+  },
+  "files": [
+    "lib"
+  ],
+  "devDependencies": {
+    "@babel/cli": "^7.4.3",
```
Comment (mjhenkes) [inline package.json]: This project shouldn't need babel since the aggregate translations script will be run by node.

File: dangerfile.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { danger, warn, fail } from 'danger';
+
+const hasCHANGELOGChanges = danger.git.modified_files.some((filePath) => {
+  const srcFilePattern = /CHANGELOG.md/i;
+  return srcFilePattern.test(filePath);
+});
+
+const hasModifiedFiles = danger.git.modified_files.some((filePath) => {
+  const filePattern = /(src)/;
```
Comment (emilyrohrbough) [inline dangerfile.js]: This should be scripts instead of src

File: package.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    "url": "https://github.com/cerner/terra-aggregate-translations/issues"
+  },
+  "browserslist": [
+    "extends browserslist-config-terra"
+  ],
+  "eslintConfig": {
+    "extends": "terra"
+  },
+  "files": [
+    "lib"
```
Comment (emilyrohrbough) [inline package.json]: I think this should be scripts

File: package.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "terra-aggregate-translations",
+  "version": "0.0.0",
+  "description": "The terra-aggregate-translations pre-build tool assists with creating the translation, intl loader and translation loader files that are configured for the specified locales. This tool is offered as a CLI script and as a setup function.",
+  "main": "lib/index.js",
+  "engines": {
+    "node": ">=8.9.2"
```
Comment (emilyrohrbough) [inline package.json]: Lets lock this down to match current engine versions in other terra repos: 
`>=8.9.2 <12`

File: package.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    "@babel/cli": "^7.4.3",
+    "@babel/core": "^7.4.3",
+    "@babel/preset-env": "^7.4.3",
+    "browserslist-config-terra": "^1.2.0",
+    "danger": "^3.7.14",
+    "eslint": "^5.14.1",
+    "eslint-config-terra": "^2.4.0",
+    "jest": "^24.1.0"
+  },
+  "scripts": {
```
Comment (emilyrohrbough) [inline package.json:40]: Lets add a `clean:install` script that removes the package-lock and node modules & then runs npm install

File: .npmignore [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+.github
+dangerfile.js
+.travis.yml
+.gitattributes
+.babelrc
+.eslintrc
+CODE_OF_CONDUCT.md
+CONTRIBUTING.md
+jestConfig.js
+jestSetup.js
```
Comment (emilyrohrbough) [inline .npmignore]: Do we have a jest setup?

File: dangerfile.js (hunk 2 above)
Comment (bjankord) [reply dangerfile.js]: Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/1730c5af56deebfe97427c6e78bea6c0ea32fb47

File: package.json (hunk 1 above)
Comment (bjankord) [reply package.json]: Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/1730c5af56deebfe97427c6e78bea6c0ea32fb47

File: .npmignore (hunk 6 above)
Comment (bjankord) [reply .npmignore]: Not anymore, it mainly had stuff for react. I didn't include it in this repo.
https://github.com/cerner/terra-aggregate-translations/pull/1/commits/e30c662bfdecaf4554f9cd135bc9552e4d933978

File: package.json (hunk 5 above)
Comment (bjankord) [reply package.json:40]: Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/e30c662bfdecaf4554f9cd135bc9552e4d933978

File: package.json (hunk 4 above)
Comment (bjankord) [reply package.json]: Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/e30c662bfdecaf4554f9cd135bc9552e4d933978

File: package.json (hunk 3 above)
Comment (bjankord) [reply package.json]: Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/e30c662bfdecaf4554f9cd135bc9552e4d933978

File: package.json [hunk 7]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-aggregate-translations",
+  "version": "0.0.0",
+  "description": "The terra-aggregate-translations pre-build tool assists with creating the translation, intl loader and translation loader files that are configured for the specified locales. This tool is offered as a CLI script and as a setup function.",
+  "main": "lib/index.js",
```
Comment (emilyrohrbough) [inline package.json]: Not sure this will be right

File: .dockerignore [hunk 8]
```diff
@@ -0,0 +1,9 @@
+.DS_Store
+.git
+.gitignore
+.idea
+log/*
+tmp
+node_modules
+lib
```
Comment (emilyrohrbough) [inline .dockerignore]: Should be able to drop `lib` throughout

File: package.json (hunk 7 above)
Comment (bjankord) [reply package.json]: We can change it once we get source code in the repo.

**Review by emilyrohrbough — APPROVED** [review]

File: .dockerignore (hunk 8 above)
Comment (bjankord) [reply .dockerignore]: Updated: https://github.com/cerner/terra-aggregate-translations/pull/1/commits/054823863b04f4d041e4dbe7fefd3dd3471d7293

File: package.json (hunk 7 above)
Comment (bjankord) [reply package.json]: Removed main key, we can add it as we add source code to the project.
https://github.com/cerner/terra-aggregate-translations/pull/1/commits/054823863b04f4d041e4dbe7fefd3dd3471d7293

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**